packages/hooks/src/useHasMFAccountDeposited.ts

```diff
@@ -0,0 +1,27 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useHasMFAccountDeposited = () => {
+    const { client } = useStore();
+    const { is_authorize } = client;
+    const expected_status = ['unwelcome', 'withdrawal_locked', 'cashier_locked'];
+
+    const hasDeposited = (status?: string[]) => {
+        return status?.some(status => expected_status.includes(status)) ?? false;
+    };
+
+    const { data } = useFetch('get_account_status', {
+        options: {
+            enabled: is_authorize,
```

Comment:
@farhan-nurzi-deriv Would it cause issues if we don't do this? 🤔 We render the component even if the user is not authorized yet? 🤔

Reply:
Because `get_account_status` requires user to be authorized first right? so i added this option. Or maybe i understand the `enabled` usage wrongly 👀

Reply:
@farhan-nurzi-deriv No you got it right 😅 I mean if the hook is only being called in a component that is only rendered after authorization then this check is not needed. It's not important so let's Ignore this comment 😅

Reply:
Oh i got your point now, thanks 🙏